bobweb/bob/command_gpt.py

```diff
@@ -112,21 +118,39 @@ def generate_and_format_result_text(self, update: Update) -> string:
         response = f'{content}\n\n{cost_message}'
         return response
 
-    def build_message(self, chat_id: int) -> List:
+    def build_message(self, chat_id: int, system_prompt: str = None) -> List:
         conversation_context = self.conversation_context.get(chat_id, [])
-        system_prompt = database.get_gpt_system_prompt(chat_id)
-
-        # System message is only added if user has specified one
+        system_prompt = system_prompt or database.get_gpt_system_prompt(chat_id)
         if system_prompt is not None:
             return [{'role': 'system', 'content': system_prompt}] + conversation_context
         else:
             return conversation_context
 
+    def handle_quick_system_prompt_sub_command(self, update: Update, command_parameter, context: CallbackContext = None):
+        sub_command = command_parameter[1]
+        sub_command_parameter = get_content_after_regex_match(command_parameter, quick_system_prompt_sub_command_regex)
 
-no_parameters_given_notification_msg = "Anna jokin syöte komennon jälkeen. '[.!/]gpt {syöte}'. Voit asettaa ChatGpt:n" \
-                                       "System-viestin komennolla '[.!/]gpt [.!/]system {uusi system viesti}'. " \
-                                       "System-viesti on kaikissa muissa viesteissä mukana menevä metatieto botille " \
-                                       "jolla voi esimerkiksi ohjeistaa halutun tyylin vastata viesteihin."
+        # If actual prompt after quick system prompt option is empty
+        if sub_command_parameter.strip() == '':
+            update.effective_message.reply_text(no_parameters_given_notification_msg)
+        else:
+            self.gpt_command(update, sub_command_parameter, context, system_prompt_id=sub_command)
+
+quick_system_prompts = {
+    "1": "You are an assistant named Bob (or Robert the Bot officially). Be brief and concise in your answers.",
+    "2": "Olet suomenkielinen apuri nimeltä Bob.",
+    "3": "You are an expert code developer with teaching skills."
+}
+
+no_parameters_given_notification_msg = f"""
+    Anna jokin syöte komennon
+    jälkeen. [.!/]gpt (syöte). Voit valita jonkin kolmesta
+    valmiista ohjeistusviestistä laittamalla numeron 1-3
+    ennen syötettä.\n
+    1: {quick_system_prompts["1"]}\n
+    2: {quick_system_prompts["2"]}\n
+    3: {quick_system_prompts["3"]}
+"""
```

Comment:
Tää ok. Tässä vois toki vielä laittaa ton koko listan tuleen automaattisesti tuolta quick_system_prompts dictistä, esim:

```suggestion
quick_system_prompts_str = ''.join([f'\n{key}: {value}' for key, value in quick_system_prompts.items()])

no_parameters_given_notification_msg = f"""
    Anna jokin syöte komennon
    jälkeen. [.!/]gpt (syöte). Voit valita jonkin kolmesta
    valmiista ohjeistusviestistä laittamalla numeron 1-3
    ennen syötettä.
    {quick_system_prompts_str}
"""
```
Tossa toki tollanen ei niin selkeä list interpolalaatio. Ei tarvi tässä muuttaa, tarjoan vain näkökulmaa.

Reply:
Katsotaan tuota sitten seuraavassa iteraatiossa.